src/app/ubs/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html

```diff
@@ -14,6 +14,7 @@ <h1 class="form__headline py-3">{{ 'address-details.title' | translate }}</h1>
               [formControlName]="getLangValue('addressRegion', 'addressRegionEng')"
               (change)="setRegionValue($event)"
               class="form-control shadow-none placeholder-positioning"
+              placeholder="Region"
```

Comment:
Maybe use here translate pipe to set value in English and Ukrainian?

Reply:
added translation